docs/use-cases/one-to-many.md

```diff
@@ -0,0 +1,138 @@
+
+When you update a fragment, it updates the configurations of all machines that use that fragment.
+
+{{< alert title="In this page" color="tip" >}}
+
+1. [Create a configuration fragment](#create-a-fragment)
+1. [Apply a fragment to multiple machines](#add-a-fragment-to-multiple-machines)
+1. [Modify an otherwise identical configuration](#modify-a-fragment)
+
+{{< /alert >}}
+
+For information on provisioning many machines, see [Provisioning](/fleet/provision/).
+
+## Create a fragment
+
+{{< table >}}
+{{% tablestep link="/configure/" %}}
+**1. Configure one machine**
+
+Start by configuring one of your machines.
+
+In the [Viam app](https://app.viam.com), use the **CONFIGURE** tab to build a configuration for all components and services you want to use on all your machines.
+
+{{<imgproc src="/use-cases/one-to-many/config.png" resize="800x" class="fill aligncenter" style="max-width: 400px" declaredimensions=true alt="Configuration builder UI">}}
+
+{{% /tablestep %}}
+{{% tablestep %}}
+**2. Copy the raw JSON**
+
+In your machine's **CONFIGURE** tab, switch to **JSON** and copy the raw JSON.
+
+{{<imgproc src="/use-cases/one-to-many/raw-json.png" resize="700x" class="fill aligncenter" style="max-width: 400px" declaredimensions=true alt="JSON subtab of the CONFIGURE tab">}}
+
+{{% /tablestep %}}
+{{% tablestep link="/fleet/fragments/" %}}
+**3. Create a fragment**
+
+Go to [app.viam.com/fragments](https://app.viam.com/fragments).
+
+Add a fragment, and paste the copied JSON configuration into it.
+
+Set your privacy settings.
+There are three options for this:
+
+- **Public:** Any user inside or outside of your organization will be able to view and use this fragment.
+- **Private:** No user outside of your organization will be able to view or use this fragment.
+- **Unlisted:** Any user inside or outside of your organization, with a direct link, will be able to view and use this fragment.
+
+Click **Save**.
+
+If you want to edit the fragment later, do it from this screen.
+
+{{<imgproc src="/use-cases/one-to-many/new-fragment.png" resize="700x" class="fill aligncenter" style="max-width: 350px" declaredimensions=true alt="app.viam.com/fragment interface">}}
+
+{{% /tablestep %}}
+{{% tablestep %}}
+{{<imgproc src="/use-cases/one-to-many/trash-can.svg" class="fill alignleft" style="max-width: 50px" declaredimensions=true alt="Delete">}}
+**3. Delete the original configuration (optional)**
+
+Now that the configuration is saved as a fragment, you can delete the original config from your machine and _replace it with the fragment_.
+In this way, you can keep all your machines up to date whenever you change the fragment.
+
+{{% /tablestep %}}
+{{< /table >}}
+
+## Add a fragment to multiple machines
+
+Now that you have a fragment, you can add it to as many machines as you'd like:
```

Comment:
Don't use now that twice so close together please

Reply:
```suggestion
With your fragment created, you can add it to as many machines as you'd like:
```